packages/amplify-graphql-relational-transformer/src/graphql-has-many-transformer.ts

```diff
@@ -177,39 +151,20 @@ export class HasManyTransformer extends TransformerPluginBase {
 
     for (const config of this.directiveList) {
       const dbType = getStrategyDbTypeFromTypeNode(config.field.type, context);
-      if (dbType === DDB_DB_TYPE) {
-        updateTableForConnection(config, context);
-      }
-      makeQueryResolver(config, context, dbType);
+      getHasManyDirectiveTransformer(dbType).generateResolvers(ctx, config);
```

Comment:
We don't need to make the change now--in fact I would recommend against it in the interest of making no functional changes--but let's remember that we'll eventually have to migrate this to strategy instead of db type, since our resource generation will be different for imported gen1 DDB tables than it is for greenfield DDB tables.

Reply:
💯. And yes, moving from db type to strategy, despite the fact that it should just work, was a bit more than I felt comfortable than I felt comfortable with for this PR.